nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/test/groovy/org/apache/nifi/processors/script/ExecuteScriptGroovyTest.groovy

```diff
@@ -149,6 +150,7 @@ class ExecuteScriptGroovyTest extends BaseScriptTest {
         }
     }
 
+    @Ignore("This test fails intermittently when the serial execution happens faster than pooled")
```

Comment:
This file should not be listed in the diff. Can you rebase against the latest master? I suspect the SHA of the commit changed from the one on your branch when I did a sign-off or a commit --amend to add the "This closes" line, or something.